From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

daveba committed:
creating new articles and updating TOC

daveba committed:
updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating doc

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new content and updating the TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
addding new files and updating TOC

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
addding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding new files and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

carolinacmoravia committed:
C14344: Extra characters that might be deleted 

Hello, @mgoedtel,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

daveba committed:
updating file content

daveba committed:
undoing change

daveba committed:
resolve merge conflicts

daveba committed:
update article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating article

daveba committed:
resolving merge conflicts

daveba committed:
created article user assigned identity to access ARM

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
update TOC

daveba committed:
updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
update article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updated TOC

daveba committed:
updating article

daveba committed:
updating article

daveba committed:
fixed merge conflicts

daveba committed:
updating article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating article

daveba committed:
updating article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating article

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
adding files

daveba committed:
adding files

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating landing page

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating landing page

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

Commit:
camfaperapp02

daveba committed:
removing unneeded articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
removing unneeded articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
update TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
removing unwanted files

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content-mvc

daveba committed:
updating file

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
update articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
updating articles

daveba committed:
updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
updates

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
update context file

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
update context file

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
article updates

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
article updates

daveba committed:
update articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
update articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
update articles

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

jmprieur committed:
redirecting active-directory/develop/active-directory-v2-devquickstarts-dotnet-web.md to the guided setup

daveba committed:
adding and updating articles

carolinacmoravia committed:
Update log-analytics-data-sources-linux-applications.md

daveba committed:
removing file

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
accepting incoming change

daveba committed:
resolving merge conflicts

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
fixing merge conflicts

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

SnehaGunda committed:
Updating text from graph to gremlin

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

SnehaGunda committed:
Adding .Net reference link for Gremlin

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

daveba committed:
adding and updating articles

eross-msft committed:
renaming files and creating new

barclayn committed:
modifying the toc to make the tutorial wrap less

Commit:
Task 985: Clarify examples and options text

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-985

barclayn committed:
editing the toc to move some articles under tutorials

martincoetzer committed:
Update concept-sspr-howitworks.md

added pass-through authentication

martincoetzer committed:
Update active-directory-aadconnectsync-installation-wizard.md

added hash and PTA

paulshealy1 committed:
Update DSVM docker details

PRMerger15 committed:
Merge pull request #38867 from martincoetzer/patch-15

Update active-directory-aadconnectsync-installation-wizard.md

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

PRMerger18 committed:
Merge pull request #38863 from martincoetzer/patch-14

Update concept-sspr-howitworks.md

daveba committed:
adding and updating articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

SnehaGunda committed:
Removing .Net link

maertendMSFT committed:
Fixing casing of clouddrive

rolyon committed:
Fix small issues

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-logs-fixes

daveba committed:
adding and updating articles

Commit:
Merge pull request #38870 from maertendMSFT/clouddrive

Fixing casing of clouddrive

barclayn committed:
updating the toc to make the tutorial entry shorter

v-shils committed:
Merge pull request #38837 from eross-msft/lizross-gdpr-includes

Renaming to remove active-directory and creating new for intro include

hpsin committed:
Add acct optional claim

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-985

Commit:
Fix merge issue of foreign file

barclayn committed:
updating the blue box on the portal

msjuergent committed:
changelink

PRMerger15 committed:
Merge pull request #38874 from hpsin/patch-9

Add acct optional claim

msjuergent committed:
changelink2

cjgronlund committed:
labs labels and links

cjgronlund committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cog-serv-edits2

Commit:
Several different updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0424

martincoetzer committed:
Update azure-ad-choose-authn.md

markingmyname committed:
[PBIE] Added embed token limit REST API page

rolyon committed:
Fixed CLI command anchor links

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-logs-fixes

PRMerger20 committed:
Merge pull request #38875 from barclayn/portal-fix

updating the blue box on the portal

Commit:
camfaperapp02

Commit:
Merge pull request #37275 from carolinacmoravia/patch-667

C14344: Extra characters that might be deleted

PRMerger10 committed:
Merge pull request #38880 from markingmyname/APBIE

[PBIE] Added embed token limit REST API page

v-shils committed:
Merge pull request #38810 from SnehaGunda/master

Updating text from graph to gremlin

v-shils committed:
Merge pull request #38852 from dksimpson/dean-985

Task 985: Clarify examples and options text

daveba committed:
making changes based on pr review

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rework-msi-content

PRMerger13 committed:
Merge pull request #38879 from martincoetzer/patch-16

Update azure-ad-choose-authn.md

rjagiewich committed:
Merge pull request #38862 from barclayn/keyvault-tocfix

Keyvault tocfix

rmca14 committed:
Merge pull request #38123 from MarkusVi/camfaperapp02

camfaperapp02

PRMerger18 committed:
Merge pull request #38872 from paulshealy1/patch-1

Update DSVM docker details

MikeDodaro committed:
Roll back changes that should have gone into release-build-cognitive-search.

rachel-msft committed:
ause, sindia

rmca14 committed:
Merge pull request #38204 from daveba/rework-msi-content

[Azure AD MSI] Rework msi content

Commit:
error03

Commit:
Merge pull request #38871 from rolyon/rolyon-logs-fixes

[Azure RBAC] Fixes to Activity logs article

rjagiewich committed:
Merge pull request #38877 from cjgronlund/cog-serv-edits2

Labs labels and doc links fixed

TrishaMc committed:
Merge pull request #38878 from Minewiskan/aas0424

Aas0424

Commit:
Merge pull request #38909 from rachel-msft/updateregions

ause, sindia

Commit:
Merge pull request #38908 from MikeDodaro/rollBackUseDisplayInclude

Roll back changes that should have gone into release-build-cognitive-…

PRMerger12 committed:
Merge pull request #38911 from MarkusVi/error03

error03

TrishaMc committed:
Merge pull request #38876 from msjuergent/wadoc

Wadoc

Brenduns committed:
formatting fix, and Acrolinx work

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #38916 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger19 committed:
Merge pull request #38914 from Brenduns/quick-fix

formatting fix, and Acrolinx work

GitHubber17 committed:
Merge pull request #38532 from jmprieur/cleanupBuild388643

redirecting ctive-directory-v2-devquickstarts-dotnet-web.md to the guided setup
